### ISSUE TITLE
bug: ERR_SSL_PROTOCOL_ERROR Keeps Popping Up in All Browsers, and Malwarebytes

### ISSUE THREAD

**Issue by AmaterasuSolar** [revoltchat/revolt#546]
### What happened?

I tried to log in for the past three days.  I keep being told this:

![image](https://github.com/revoltchat/revolt/assets/135478431/5904c9c3-2a5f-4be7-aab6-5be6a022879e)

Is this being resolved?  Please email Me at AmaterasuSolar@protonmail.com when it is fixed.

**Comment by AmaterasuSolar** [comment]
This is what I get from Malwarebytes:



![Image](https://github.com/revoltchat/revolt/assets/135478431/3171de9d-4b05-48a0-821c-0925e3b4b0c0)

**Comment by insertish** [comment]
I've created a post on their forum, waiting to hear back now.

**Comment by insertish** [comment]
Just following up here, their claim was legitimate in terms of, there was a file uploaded by a previously banned user (whose data had not been deleted yet, I think this is the person who was DDoSing us last week).

The content has been removed so hopefully they'll unblock it on their end soon, not sure why they decided to block beyond autumn.revolt.chat, I have inquired about that too.

Follow-up issue: https://github.com/revoltchat/autumn/issues/27

**Comment by AmaterasuSolar** [comment]
Not sure what "Closed #546 as completed" means - but I guess it just means on Github?

a

"What Human does not deserve leisure and industry of interest, if (S)He chooses all behavior Ethically?"
"Revolution in ideas, not blood."

♥♥♥

"Did You give an oath and find it's bait and switch? Well, there is no oath then, is there?"
"ALL money systems promote the most psychopathic to the top of the money/power heap - THEY will do ANYTHING to get there."
"The ONLY tool They use is money, and I say We can put that down and pick up far better tools."
"The love of money is the root of all evil; remove the soil in which the root grows..."
"If the universe is made of mostly "dark" energy...can We use it to run Our cars?"
"If You want peace, take the PROFIT out of war."
.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by insertish** [comment]
MalwareBytes have already confirmed they're removing us from the block list, this will take some time to propagate.

**Comment by AmaterasuSolar** [comment]
Okay. I will be patient! Thanks again!!!

a

"What Human does not deserve leisure and industry of interest, if (S)He chooses all behavior Ethically?"
"Revolution in ideas, not blood."

♥♥♥

"Did You give an oath and find it's bait and switch? Well, there is no oath then, is there?"
"ALL money systems promote the most psychopathic to the top of the money/power heap - THEY will do ANYTHING to get there."
"The ONLY tool They use is money, and I say We can put that down and pick up far better tools."
"The love of money is the root of all evil; remove the soil in which the root grows..."
"If the universe is made of mostly "dark" energy...can We use it to run Our cars?"
"If You want peace, take the PROFIT out of war."
.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by AmaterasuSolar** [comment]
Just a quick note to let You know, I'm back in! Thanks so very much!

a

"What Human does not deserve leisure and industry of interest, if (S)He chooses all behavior Ethically?"
"Revolution in ideas, not blood."

♥♥♥

"Did You give an oath and find it's bait and switch? Well, there is no oath then, is there?"
"ALL money systems promote the most psychopathic to the top of the money/power heap - THEY will do ANYTHING to get there."
"The ONLY tool They use is money, and I say We can put that down and pick up far better tools."
"The love of money is the root of all evil; remove the soil in which the root grows..."
"If the universe is made of mostly "dark" energy...can We use it to run Our cars?"
"If You want peace, take the PROFIT out of war."
.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
